command/rootchain/fund/fund.go

```diff
@@ -122,12 +158,33 @@ func runCommand(cmd *cobra.Command, _ []string) {
 					return err
 				}
 
-				lock.Lock()
-
-				resList = append(resList, &result{
+				result := &result{
 					ValidatorAddr: validatorAcc,
 					TxHash:        types.Hash(receipt.TransactionHash),
-				})
+				}
+
+				if params.mintRootToken {
+					// mint tokens to depositor, so he is able to send them
+					mintTxn, err := helper.CreateMintTxn(validatorAcc, validatorAcc, rootTokenAddr, ethgo.Ether(100))
```

Comment:
Extract `ethgo.Ether(100)` to a local variable and use it both here and for fund transaction?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1446/commits/5baa9f550555c823722fe79c62f2c9d3077c8954